### PR TITLE
2 packages from OCamlPro/ocp-index at 1.3.0

### PR DESCRIPTION
**Pull request by AltGr**
This pull-request concerns:
-`ocp-browser.1.3.0`: Console browser for the documentation of installed OCaml libraries
-`ocp-index.1.3.0`: Lightweight completion and documentation browsing for OCaml libraries

---
* Homepage: http://www.typerex.org/ocp-index.html
* Source repo: git+https://github.com/OCamlPro/ocp-index.git
* Bug tracker: https://github.com/OCamlPro/ocp-index/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/ocp-browser/ocp-browser.1.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "Gabriel Radanne"
+]
+homepage: "http://www.typerex.org/ocp-index.html"
+bug-reports: "https://github.com/OCamlPro/ocp-index/issues"
+license: "GPL-3.0-only"
+tags: [ "org:ocamlpro" "org:typerex" ]
+dev-repo: "git+https://github.com/OCamlPro/ocp-index.git"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "ocaml" {>= "4.02.0"}
```
Comment (mseri) [inline packages/ocp-browser/ocp-browser.1.3.0/opam:19]:
```suggestion
  "ocaml" {>= "4.02.0" & < "4.13~"}
```

Comment (AltGr) [reply packages/ocp-browser/ocp-browser.1.3.0/opam:19]: Indeed! Thanks

File: packages/ocp-index/ocp-index.1.3.0/opam [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  "Gabriel Radanne"
+]
+homepage: "http://www.typerex.org/ocp-index.html"
+bug-reports: "https://github.com/OCamlPro/ocp-index/issues"
+license: ["LGPL-2.1-only with OCaml-LGPL-linking-exception" "GPL-3.0-only"]
+tags: [ "org:ocamlpro" "org:typerex" ]
+dev-repo: "git+https://github.com/OCamlPro/ocp-index.git"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "ocaml" {>= "4.02.0"}
```
Comment (mseri) [inline packages/ocp-index/ocp-index.1.3.0/opam:24]:
```suggestion
  "ocaml" {>= "4.02.0" & < "4.13~"}
```

Oops, I forgot it here

**Comment by mseri** [comment]
The remaining failures are due to incorrect bounds in ocp-build. Do you know what we should use as lower bound?

**Comment by AltGr** [comment]
Huh ? The package hasn't been using `ocp-build` for a while and the PR doesn't mention it ?
Ah, the error might come from trying to install an old version of `ocp-indent` which relied on `ocp-build` ?

**Comment by AltGr** [comment]
Not directly related, but do you know why `opam-ci` doesn't use the package cache and relies on the upstream archives for all dependencies ?
It could be done by just adding `archive-mirrors: "https://opam.ocaml.org/cache"` to `~/.opam/config`

**Comment by mseri** [comment]
Yes, it happens in the lower-bound test:
```
# Run eval $(opam env) to update the current shell environment
The following actions will be performed:
  - remove    mew_vi            0.5.0          [conflicts with dune]
  - remove    opam-depext       1.1.5
  - install   conf-m4           1              [required by ocamlfind]
  - install   jbuilder          1.0+beta20.1   [required by camomile, cppo]
  - remove    ocplib-endian     1.1            [conflicts with cppo, dune]
  - remove    dune-configurator 2.8.5          [conflicts with dune, result]
  - remove    mew               0.1.0          [conflicts with dune, result]
  - downgrade ocamlfind         1.9.1 to 1.8.0
  - downgrade camomile          1.0.2 to 1.0.1
  - remove    csexp             1.5.1          [conflicts with dune]
  - downgrade result            1.5 to 1.0
  - remove    trie              1.0.0          [conflicts with dune]
  - recompile base-bytes        base           [uses ocamlfind]
  - downgrade react             1.2.1 to 1.0.0 [uses ocamlfind]
  - downgrade topkg             1.0.3 to 0.8.1 [uses ocamlfind, result]
  - downgrade dune              2.8.5 to 2.0.1
  - downgrade cppo              1.6.7 to 1.6.1
  - downgrade cmdliner          1.0.4 to 1.0.0 [uses ocamlfind, result]
  - recompile charInfo_width    1.1.0          [uses dune, camomile, result]
  - downgrade seq               base to 0.2.2  [uses dune]
  - downgrade mmap              1.1.0 to 1.0.2 [uses dune]
  - downgrade zed               3.1.0 to 2.0.1 [uses dune, camomile, result]
  - recompile re                1.9.0          [uses dune]
  - downgrade lwt               5.4.1 to 4.2.0 [uses dune, cppo, result]
  - install   ocp-build         1.99.21        [required by ocp-indent]
  - downgrade lwt_react         1.1.4 to 1.0.1 [uses lwt]
  - downgrade lwt_log           1.1.1 to 1.1.0 [uses jbuilder]
  - downgrade ocp-indent        1.8.1 to 1.6.1 [upstream or system changes]
  - downgrade lambda-term       3.1.0 to 2.0   [uses camomile, dune]
  - recompile ocp-index         1.3.0          [uses cppo, dune]
===== 3 to install | 4 to recompile | 16 to downgrade | 7 to remove =====
```

**Comment by mseri** [comment]
> Not directly related, but do you know why `opam-ci` doesn't use the package cache and relies on the upstream archives for all dependencies ?
> It could be done by just adding `archive-mirrors: "https://opam.ocaml.org/cache"` to `~/.opam/config`

No idea, ping @kit-ty-kate 
It is useful to check that packages are still retrievable and did not change though

**Comment by AltGr** [comment]
I took the opportunity to ping the maintainer of `ocp-build` :)
(but maintain it's unrelated to my PR ;))

**Comment by kit-ty-kate** [comment]
https://github.com/ocaml/opam-repository/pull/18946 should fix the lower-bounds issues. I'll rerun them once the fix is merged

**Comment by kit-ty-kate** [comment]
Thanks! Could you return the fixes to the opam files upstream?

**Pull request merged**